src/test/java/com/salesforce/dataloader/process/ProcessExtractTestBase.java

```diff
@@ -218,15 +218,9 @@ protected void runTestSoqlWithTableNameInSelect() throws ProcessInitializationEx
         final ExtractContactGenerator contactGenerator = new ExtractContactGenerator();
         final String soql = contactGenerator.getSOQL("Contact.Id, Contact.Account.id");
         final Map<String, String> argmap = getTestConfig(soql, "Contact", false);
-        if (isBulkAPIEnabled(argmap)) {
```

Comment:
Any reason why you deleted the if block?

Reply:
The if block contained code to verify that the Bulk API call resulted in an error message. The Bulk API call succeeds because it supports relationship lookup. Consequently, I have removed the if block so that the Bulk API call results are processed in the same way as Partner API call results.